cmake/dependencies/CMakeLists.txt

```diff
@@ -213,3 +213,39 @@ if(BUILD_Cbc)
 endif()
 
 set(CMAKE_CXX_STANDARD 17)
+
+# ##############################################################################
+# GLPK
+# ##############################################################################
+if(BUILD_GLPK)
+  message(CHECK_START "Fetching GLPK")
+  list(APPEND CMAKE_MESSAGE_INDENT "  ")
+
+  find_program(MAKE_EXE NAMES gmake nmake make)
```

Comment:
why you need this ?

I means if we are here we already have a working generator so we use the actual one using `${CMAKE_GENERATOR}`

Reply:
I use it to give feedback for a specific error code lower down - if the GLPK build needs access to `make`, the user has the option of changing the perms on the specific binary used rather than run the build as root. I'll move it closer for clarity.